woocommerce-pdf-invoices-packingslips.php

```diff
@@ -273,12 +272,19 @@ public function need_woocommerce() {
 	/**
 	 * PHP version requirement notice
 	 */
-	
 	public function required_php_version() {
-		$error = __( 'WooCommerce PDF Invoices & Packing Slips requires PHP 5.3 or higher (5.6 or higher recommended).', 'woocommerce-pdf-invoices-packing-slips' );
-		$how_to_update = __( 'How to update your PHP version', 'woocommerce-pdf-invoices-packing-slips' );
-		$message = sprintf('<div class="error"><p>%s</p><p><a href="%s">%s</a></p></div>', $error, 'http://docs.wpovernight.com/general/how-to-update-your-php-version/', $how_to_update);
-	
+		$error_message	= __( 'WooCommerce PDF Invoices & Packing Slips requires PHP 5.6 or higher (7.1 or higher recommended).', 'woocommerce-pdf-invoices-packing-slips' );
+		$php_message	= __( 'We strongly recommend to %supdate your PHP version%s.', 'woocommerce-pdf-invoices-packing-slips' );
+		$add_on_message	= __( 'If you cannot upgrade your PHP version, you can download %sthis addon%s to enable backwards compatibility with PHP5.6.', 'woocommerce-pdf-invoices-packing-slips' );
+
+		$message = '<div class="error">';
+		$message .= sprintf( '<p>%s</p>', $error_message );
+		$message .= sprintf( '<p>'.$php_message.'</p>', '<a href="https://docs.wpovernight.com/general/how-to-update-your-php-version/" target="_blank">', '</a>' );
+		if ( version_compare( PHP_VERSION, '5.6', '>' ) ) {
+			$message .= sprintf( '<p>'.$add_on_message.'</p>', '<a href="https://github.com/wpovernight/wcpdf-dompdf-0.8.3/releases" target="_blank">', '</a>' );
```

Comment:
this links to a page that says:

> There aren’t any releases here

Maybe we should even provide a direct link to the release zip instead?

Reply:
Thinking more about this, I think we should actually link to a page in our documentation instead, where we can always link to the latest version and provide more detailed instructions on installing the extension. We need to make this transition as smooth as possible.